web/frontend/src/pages/election/Show.tsx

```diff
@@ -206,7 +235,7 @@ const ElectionShow: FC = () => {
             <div className="font-bold uppercase text-lg text-gray-700 pb-2">{t('action')}</div>
             <div className="px-2">
               {DKGLoading && <LoadingButton>{t('actionLoading')}</LoadingButton>}{' '}
```

Comment:
You forgot about this one 😬
``DKGLoading && ongoingAction === OngoingAction.None &&``